tests/test_evaluation.py

```diff
@@ -134,6 +138,10 @@ def test_evaluate_factuality_oracle_no_skips():
     assert agg_metrics["entities"]["Non-hallucinated"] == 177
     assert agg_metrics["entities"]["total"] == 98 + 16 + 23 + 177
 
+    # This isn't exactly 1 because we only run 5 iterations, and it's possible
+    # to stil no have fully factual entities after 5 iterations.
```

Comment:
This is not 1 because instead of skipping we include non-factual hallucinations

Reply:
true comment is bad, i'll delete it.